res/EntityFactory.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "id":0,
+    "name": "Default",
+    "hitbox": [0,0,0,0],
+    "position": [0,0]
```

Comment:
No comma. Might be a problem.

Reply:
What makes you think that it will be a problem? There is no need for comma if it's last.

Reply:
Oh, sorry nevermind. Haven't seen last line there :P